pkgs/development/tools/agda-pkg/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, python3Packages, agda }:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "agda-pkg";
+  version = "0.1.49";
```

newer version in available
```suggestion
  version = "0.1.50";
```